package.json

```diff
@@ -23,27 +23,44 @@
   "license": "BSD-3-Clause",
   "devDependencies": {
     "array.prototype.find": "^1.0.0",
-    "babel-core": "^6.5.2",
-    "babel-loader": "^6.2.3",
-    "babel-preset-react": "^6.5.0",
+    "babel-core": "^6.4.0",
+    "babel-loader": "^6.2.1",
+    "babel-polyfill": "^6.3.14",
+    "babel-preset-react": "^6.3.13",
+    "babel-register": "^6.3.13",
     "chai": "^3.0.0",
-    "chai-spies": "^0.7.1",
     "d3": "^3.5.6",
     "d3-geo-projection": "^0.2.15",
     "datalib": "^1.4.6",
     "deep-diff": "^0.3.3",
+    "enzyme": "^1.2.0",
+    "es6-promise": "^3.0.2",
     "es6-promisify": "^3.0.0",
     "eslint": "^1.10.3",
     "eslint-loader": "^1.2.1",
     "eslint-plugin-react": "^3.16.1",
+    "fs": "0.0.2",
     "istanbul": "^0.3.21",
     "jshint": "^2.8.0",
+    "json-loader": "^0.5.4",
+    "jstransform": "^11.0.3",
```

Comment:
This is another one that was removed in #266; it's unclear whether it was added back in to support this build or due to a merge